Common/src/gRPC/Services/Submitter.cs

```diff
@@ -672,6 +672,12 @@ await payloadUploadTasks.WhenAll()
                                  cancellationToken)
                     .ConfigureAwait(false);
 
+    await resultTable_.SetTaskOwnership(sessionId,
```

Comment:
I don't understand why it was not there before, and what changed to mandate its use now.

Reply:
Beforehand, we were creting the result during task creation so we were creating the result with the field already filled.